code/game/objects/items.dm

```diff
@@ -212,6 +212,18 @@ GLOBAL_DATUM_INIT(welding_sparks, /mutable_appearance, mutable_appearance('icons
 	/// Used in obj/item/examine to determines whether or not to detail an item's statistics even if it does not meet the force requirements
 	var/override_notes = FALSE
 
+/obj/item/New()
```

Comment:
Why New()?

Reply:
It NEEDS to exist as soon as the object does. Sadly, I don't know of a better way to do this. `alt_appearances_by_bodytype` needs to exist (if it has a value), because it can be referenced before Initialize(), example: Character Creation.

Reply:
> It NEEDS to exist as soon as the object does

Why?

Also if this is true then this needs to be documented as precisely as possible so that:

1. It doesn't look like a mistake
2. If assumptions change, we can change this to Initialize

Reply:
> > It NEEDS to exist as soon as the object does
> 
> Why?

`alt_appearances_by_bodytype needs to exist (if it has a value), because it can be referenced before Initialize(), example: Character Creation.`

Though, good idea. It should be explicit why it's there.

Reply:
> example: Character Creation.

This isn't specific enough, what exactly is created during character creation. Preferences running before atoms SS is a known issue, but that gets resolved by calling forcing specific things to initialize immediately, for instance.

Reply:
> > example: Character Creation.
> 
> This isn't specific enough, what exactly is created during character creation? Preferences running before atoms SS is a known issue, but that gets resolved by calling forcing specific things to initialize immediately, for instance.
Clothing.

From what little I know, `INITIALIZE_IMMEDIATE()` is only applied to Organs (for this purpose, anyway). It would be silly to immediately initialize all items.

Reply:
I didn't mean all items, I mean to figure out if it's viable to tweak it for the specific clothing that's actually used

If it's all clothing that sucks but reasonable

Reply:
Any item can be worn and could theoretically have a unique sprite based on a bodytype. Dev UX was a huge reason for the bodytype refactor. The old code sucked to touch.

Reply:
Right